core/src/commonMain/kotlin/com/ustadmobile/core/controller/SessionDetailListPresenter.kt

```diff
@@ -0,0 +1,66 @@
+package com.ustadmobile.core.controller
+
+import com.ustadmobile.core.util.SortOrderOption
+import com.ustadmobile.core.view.SessionDetailListView
+import com.ustadmobile.core.view.SessionsListView.Companion.ARG_CONTEXT_REGISTRATION
+import com.ustadmobile.core.view.UstadView
+import com.ustadmobile.door.DoorLifecycleOwner
+import com.ustadmobile.door.doorMainDispatcher
+import com.ustadmobile.lib.db.entities.StatementWithSessionDetailDisplay
+import com.ustadmobile.lib.db.entities.UmAccount
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import org.kodein.di.DI
+
+class SessionDetailListPresenter(context: Any, arguments: Map<String, String>,
+                            view: SessionDetailListView,
+                            di: DI, lifecycleOwner: DoorLifecycleOwner)
+    : UstadListPresenter<SessionDetailListView, StatementWithSessionDetailDisplay>(
+        context, arguments, view, di, lifecycleOwner) {
+
+    private var contentEntryUid: Long = 0L
+    private var selectedPersonUid: Long = 0L
+    private var selectedSession: String = ""
+
+    override suspend fun onCheckAddPermission(account: UmAccount?): Boolean {
+        return false
+    }
+
+    override suspend fun onLoadFromDb() {
+        super.onLoadFromDb()
+        contentEntryUid = arguments[UstadView.ARG_ENTITY_UID]?.toLong() ?: 0
```

Comment:
ARG_ENTITY_UID is to for when we are passing the uid of a specific entity to load. That isn't really what is happening here (we are actually filtering a list by content entry, person, and session). Please change to using ARG_CONTENT_ENTRY_UID